docker/CPU/Dockerfile

```diff
@@ -43,12 +43,14 @@ RUN curl -fsSL -v -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/Mini
     /opt/conda/bin/conda install -y python=${PYTHON_VERSION} conda-build pyyaml numpy ipython cython typing typing_extensions mkl mkl-include ninja && \
     /opt/conda/bin/conda clean -ya
 
-### pytorch
-RUN /opt/conda/bin/pip install torch==${TORCH_VERSION}+cpu torchvision==${TORCHVISION_VERSION}+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html
-ENV PATH /opt/conda/bin:$PATH
-
 ### install open-mim
-RUN /opt/conda/bin/pip install mmcv-full==${MMCV_VERSION} -f https://download.openmmlab.com/mmcv/dist/cpu/torch${TORCH_VERSION}/index.html
+RUN /opt/conda/bin/pip install --no-cache-dir openmim
+
+### pytorch mmcv onnxruntime and openvino
+RUN mim install torch==${TORCH_VERSION}+cpu torchvision==${TORCHVISION_VERSION}+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html \
+    && mim install --no-cache-dir "mmcv>=2.0.0rc2,<2.1.0" onnxruntime==${ONNXRUNTIME_VERSION} openvino-dev
```

Comment:
why using `<2.1.0`

Reply:
may use `MMCV_VERSION` to control mmcv version.

Reply:
How about now

Reply:
`>=2.0.0rc2` could be the default value for `MMCV_VERSION`

Reply:
FYI, the following are default values, but users can pass other values when building images.
```
ARG TORCH_VERSION=1.10.0
ARG TORCHVISION_VERSION=0.11.0
ARG ONNXRUNTIME_VERSION=1.8.1
```

Reply:
How to deal with it when i set `MMENGINE_VERSION` == 0.3.0
```dockerfile
ARG MMENGINE_VERSION=">=0.3.0"
"mmengine"${MMENGINE_VERSION}
```

Reply:
`MMENGINE_VERSION` and `MMCV_VERSION` will be set to specific ones like other `ARGS` in the future. This is a temporal solution.